proparse/src/main/java/org/prorefactor/proparse/antlr4/TokenList.java

```diff
@@ -8,63 +8,62 @@
  * Contributors:
  *    John Green - initial API and implementation and/or initial documentation
  *******************************************************************************/ 
-package org.prorefactor.proparse;
+package org.prorefactor.proparse.antlr4;
 
 import java.io.StringWriter;
 import java.util.ArrayList;
 import java.util.List;
 
-import org.prorefactor.core.NodeTypes;
-import org.prorefactor.core.ProToken;
+import org.antlr.v4.runtime.CharStream;
+import org.antlr.v4.runtime.IntStream;
+import org.antlr.v4.runtime.Token;
+import org.antlr.v4.runtime.TokenFactory;
+import org.antlr.v4.runtime.TokenSource;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import antlr.Token;
-import antlr.TokenStream;
-import antlr.TokenStreamException;
-
 /**
- * List of tokens for input to the parser. This class is responsible for gathering a list of tokens from the postlexer,
- * and examining that list for fine-tuning before sending it on to the parser. The complete tool chain is:<ol>
- * <li>preprocessor 
- * <li>lexer
- * <li>postlexer
- * <li>tokenlist
- * <li>filter
+ * List of tokens for input to the parser. This class is responsible for gathering a list of tokens from the PostLexer,
+ * and examining that list for fine-tuning before sending it on to the parser.
+ * 
+ * The complete tool chain is:<ol>
+ * <li>Lexer 
+ * <li>PostLexer
+ * <li>TokenList
  * <li>parser
  * </ol>
+ * 
+ * TODO The temporary list should be avoided
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1135)